flink-kubernetes/src/test/java/org/apache/flink/kubernetes/utils/KubernetesUtilsTest.java

```
@@ -75,6 +81,91 @@ public void testCheckWithFixedPort() {
         testCheckAndUpdatePortConfigOption("6123", "16123", "6123");
     }
 
+    @Test
+    public void testLoadPodFromTemplateWithNonExistPathShouldReturnEmptyPod() {
```

Maybe it is better to fail hard.